pkgs/top-level/all-packages.nix

```diff
@@ -2654,7 +2654,11 @@ with pkgs;
   mainsail = callPackage ../applications/misc/mainsail { };
 
   # Does not build with default Haskell version because upstream uses a newer Cabal version.
-  mailctl = haskell.packages.ghc94.callPackage ../tools/networking/mailctl { };
+  mailctl = (haskell.packages.ghc94.callPackage ../tools/networking/mailctl {}).overrideScope (final: prev: {
+    # Dependency twain requires an older version of http2, and we cannot safely
```

Comment:
I would remove the "safely".

Reply:
Oh, and unless there is a good reason prefer using haskellPackages over haskell.packages.ghc94 so that this override does not go stale.

Reply:
Done and done.